build/docker/templates/intel-dlb-initcontainer.Dockerfile.in

```diff
@@ -0,0 +1,9 @@
+FROM debian:unstable
```

Comment:
btw, distroless with toybox should work right?

Reply:
+1 for using distroless

Reply:
sure, let me change to toybox!

Reply:
shellcheck complains about "echo | sed" and if I do as it suggested, then it does not work in toybox.
So, i just left that one unsolved.
We need to choose either using another distro or just give up clearing shellcheck.

Reply:
maybe try:
```
# export FOO=virtfn42
# echo ${FOO#virtfn}
42
```

Reply:
@mythi Thanks!